src/dropbox.js

```diff
@@ -50,6 +51,8 @@ const b64 = typeof btoa === 'undefined'
  * should only be used for testing as scaffolding to avoid making network requests.
  * @arg {String} [options.domainDelimiter] - A custom delimiter to use when separating domain from
  * subdomain. This should only be used for testing as scaffolding.
+ * @arg {Object} [options.customHeaders] - An object designed to set custom headers to use
```

Comment:
This should have some indication of what format this object should be in

Reply:
This also needs to be reflected in the TS types file or TS clients won't be able to use this